cartoframes/context.py

```diff
@@ -473,6 +473,67 @@ def sync(self, dataframe, table_name):
         """
         pass
 
+    def fetch(self, query, decode_geom=False, query_columns=None):
+        """Pull the result from an arbitrary SQL query from a CARTO account
```

Comment:
We should probably stress that this method is for `SELECT` queries and another method is for `INSERT`s and `UPDATE`s

Reply:
Good catch 👍